src/view/writer.js

```diff
@@ -12,704 +12,993 @@ import ContainerElement from './containerelement';
+	 * @returns {module:engine/view/containerelement~ContainerElement} Created element.
+	 */
+	createContainerElement( name, attributes ) {
+		return new ContainerElement( name, attributes );
+	}
 
-/**
- * Merges {@link module:engine/view/attributeelement~AttributeElement attribute elements}. It also merges text nodes if needed.
- * Only {@link module:engine/view/attributeelement~AttributeElement#isSimilar similar} attribute elements can be merged.
- *
- * In following examples `<p>` is a container and `<b>` is an attribute element:
- *
- *		<p>foo[]bar</p> -> <p>foo{}bar</p>
- *		<p><b>foo</b>[]<b>bar</b></p> -> <p><b>foo{}bar</b></p>
- *		<p><b foo="bar">a</b>[]<b foo="baz">b</b></p> -> <p><b foo="bar">a</b>[]<b foo="baz">b</b></p>
- *
- * It will also take care about empty attributes when merging:
- *
- *		<p><b>[]</b></p> -> <p>[]</p>
- *		<p><b>foo</b><i>[]</i><b>bar</b></p> -> <p><b>foo{}bar</b></p>
- *
- * **Note:** Difference between {@link module:engine/view/writer~writer.mergeAttributes mergeAttributes} and
- * {@link module:engine/view/writer~writer.mergeContainers mergeContainers} is that `mergeAttributes` merges two
- * {@link module:engine/view/attributeelement~AttributeElement attribute elements} or {@link module:engine/view/text~Text text nodes}
- * while `mergeContainer` merges two {@link module:engine/view/containerelement~ContainerElement container elements}.
- *
- * @see module:engine/view/attributeelement~AttributeElement
- * @see module:engine/view/containerelement~ContainerElement
- * @see module:engine/view/writer~writer.mergeContainers
- * @function module:engine/view/writer~writer.mergeAttributes
- * @param {module:engine/view/position~Position} position Merge position.
- * @returns {module:engine/view/position~Position} Position after merge.
- */
-export function mergeAttributes( position ) {
-	const positionOffset = position.offset;
-	const positionParent = position.parent;
+	/**
+	 * Creates new {@link module:engine/view/editableelement~EditableElement}.
+	 *
+	 *		writer.createEditableElement( document, 'paragraph' );
+	 *		writer.createEditableElement( document, 'paragraph', { 'alignment': 'center' } );
+	 *
+	 * @param {module:engine/view/document~Document} document View document.
+	 * @param {String} name Name of the element.
+	 * @param {Object} [attributes] Elements attributes.
+	 * @returns {module:engine/view/editableelement~EditableElement} Created element.
+	 */
+	createEditableElement( name, attributes ) {
+		const editableElement = new EditableElement( name, attributes );
+		editableElement.document = this.document;
+
+		return editableElement;
+	}
 
-	// When inside text node - nothing to merge.
-	if ( positionParent.is( 'text' ) ) {
-		return position;
+	/**
+	 * Creates new {@link module:engine/view/emptyelement~EmptyElement}.
+	 *
+	 *		writer.createEmptyElement( 'paragraph' );
+	 *		writer.createEmptyElement( 'paragraph', { 'alignment': 'center' } );
```

Comment:
Paragraph is not an empty element, so the example is wrong. You can use `img` here.